libs/langchain/langchain/chat_models/anyscale.py

```diff
@@ -137,8 +132,8 @@ def validate_environment_override(cls, values: dict) -> dict:
         try:
             if is_openai_v1():
                 client_params = {
-                    "api_key": values["openai_api_key"],
-                    "base_url": values["openai_api_base"],
+                    "api_key": values["anyscale_api_key"].get_secret_value(),
```

Comment:
Looks like this might correspond with some changes in `Anyscale` that won't be available here in `ChatAnyscale`

Reply:
The point is to remove `openai_` prefixed variables in `anyscale.py`. 
And `anyscale_api_key` is SecreStr so we need to call `.get_secret_values()` explicitly to get the value.

Reply:
The previous code is hacky ( line 105-109). `openai_api_key` are still simple strings and used later in the code. `anyscale_api_key` is secretstr but not used. 
Now the new version fixed it